vcluster/_fragments/integrations/cert-manager.mdx

```diff
@@ -0,0 +1,197 @@
+
+### Prerequisites
+
+<BasePrerequisites />
+
+- `cert-manager` operator installed on your host cluster. See instructions at https://cert-manager.io
+
+## Cert manager integration
+
+To enable the cert-manager integration, set the following fields as shown below:
+
+
+```yaml title="Enable cert-manager integration"
+integrations:
+  certManager:
+    enabled: true
+```
+
+This configuration enables the integration, imports cluster-scoped ClusterIssuers from the host cluster into the virtual cluster, and exports namespaced Issuers and Certificates from the virtual cluster into the host cluster.
+
+<Flow id="cert-manager-integration">
+ <Step>
+
+  Create the Issuer
+
+Create a <Label>file</Label> named `issuer.yaml` with the following content:
+
+```yaml title="ClusterIssuer configuration"
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-staging
+spec:
+  acme:
+    # highlight-start
+    # Replace this email address with your own.
+    # Let's Encrypt will use this to contact you about expiring
+    # certificates, and issues related to your account.
+    # highlight-end
+    email: user@example.com
+    server: https://acme-staging-v02.api.letsencrypt.org/directory
+    privateKeySecretRef:
+      # Secret resource that will be used to store the account's private key.
+      name: example-issuer-account-key
+    # Add a single challenge solver, HTTP01 using nginx
+    solvers:
+    - http01:
+        ingress:
+          ingressClassName: nginx
+```
+
+</Step>
+<Step>
+Apply the Issuer configuration:
+
+```bash title="Apply the Issuer configuration"
+kubectl apply -f issuer.yaml
+```
+:::note
+This creates a corresponding Issuer in the host cluster.
+:::
+
+</Step>
+<Step>
+
+Create and apply Issuer and Certificate
+
+After the virtual cluster is running, create an Issuer and Certificate inside the virtual cluster. This guide uses a `letsencrypt-staging` issuer for demonstration purposes.
+
+</Step>
+
+<Step>
+Create the Certificate
+
+Create a file named `certificate.yaml` with the following content:
+
+```yaml title="Certificate configuration"
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: quickstart-example-tls
+  namespace: default
+spec:
+  dnsNames:
+  - example.example.com
+  issuerRef:
+    group: cert-manager.io
+    kind: Issuer
+    name: letsencrypt-staging
+  secretName: quickstart-example-tls
+  usages:
+  - digital signature
+  - key encipherment
+```
+</Step>
+<Step>
+Apply the Certificate configuration:
+
+```bash title="Apply the Certificate configuration"
+kubectl apply -f certificate.yaml
+```
+
+:::tip
+After the certificate is created in the virtual cluster, the integration syncs the created secret back to the virtual cluster after the cert-manager operator creates it in the host cluster.
+:::
+
+</Step>
+</Flow>
+
+## Verify the integration
+
+<Flow id="verify-integration">
+  <Step>
+Check the status of the Issuer:
+
+   ```bash title="Check Issuer status"
```

Comment:
Where are we checking this? In the host or virtual cluster?

Reply:
I've added clarifications on which cluster we check.